src/test/java/com/crawler/extractor/CrawlerConfServiceTest.java

```diff
@@ -0,0 +1,104 @@
+package com.crawler.extractor;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.webAppContextSetup;
+
+import java.nio.charset.Charset;
+import java.util.Date;
+
+import org.bson.types.ObjectId;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.context.WebApplicationContext;
+
+import com.crawler.extractor.model.Crawler;
+import com.crawler.extractor.model.CrawlerConf;
+import com.crawler.extractor.model.Status;
+import com.crawler.extractor.repository.ICrawlerConfRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class CrawlerConfServiceTest {
+	
+	private final String URL_TO_UPDATE_CRAWLER_CONF = "extractor/rest/v1/crawler/conf";
+	
+	@LocalServerPort
+	private int port;
+
+	private MediaType contentType = new MediaType(MediaType.APPLICATION_JSON.getType(),
+			MediaType.APPLICATION_JSON.getSubtype(), Charset.forName("utf8"));
+
+	private MockMvc mockMvc;
+
+	private CrawlerConf crawlerConf;
+
+	@Autowired
+	private WebApplicationContext webApplicationContext;
+	
+	@Autowired
+	private ICrawlerConfRepository iCrawlerConfRepository;
+
+	private String mapToJson(Object object) throws JsonProcessingException {
+		ObjectMapper objMapper = new ObjectMapper();
+		return objMapper.writeValueAsString(object);
+	}
+	
+	@Before
+	public void setup() throws Exception {
+		mockMvc = webAppContextSetup(webApplicationContext).build();
+		crawlerConf = new CrawlerConf();
+		crawlerConf.setId(new ObjectId());
+		crawlerConf.setMaxNumberActiveCrawler(4);
+		crawlerConf.setCreatedDate(new Date());
```

Comment:
it is better to create one date and use it for createdDate and modified date, but for it you have to change second call for `setCreatedDate` to `setModifiedDate`. also you can create constructor for CrawlerConf and use it instead of calling set for each field

Reply:
I create a class variable as date, and use it for setting createdDate and ModifiedDate.